landlab/components/flow_director/flow_direction_dinf.py

```diff
@@ -161,8 +161,8 @@ def flow_directions_dinf(grid,
     triangle_neighbors_at_node = triangle_neighbors_at_node.swapaxes(0,1)
 
     # next create, triangle links at node
-    l_at_node = grid.links_at_node
-    dl_at_node = grid._diagonal_links_at_node
+    l_at_node = grid.d8_at_node[:, :4]
+    dl_at_node = grid.d8_at_node[:, 4:]
```

Comment:
My second comment. As a long time attempter of the NYT crossword puzzle, I always appreciate the consistency tense and singular/plural between the clue and the answer in the puzzle. 

My impression is that landlab is pretty good about being consistent in function and attribute names. I know that this helped me as I learned landlab because it helped me guess what function names might be. 

This begs the question: is d8 singular, or both plural and singular? In the replaced code, links was plural and nodes is often plural; so, should d8 be d8s for consistency?

I've not made notes of this everywhere in the PR but instead have attempted to list all the functions/attributes related to this issue here:

For diagonals
```python
grid. length_of_diagonal
grid. status_at_diagonal
```

For d8(s):
```python
grid.number_of_d8
grid.d8_at_node
grid. length_of_d8
grid. status_at_d8
grid. active_d8
```

Reply:
I think `d8` is singular. The plural should be `d8s`. So `d8s_at_node`, not `d8_at_node`. This was something I missed the first time through but wanted to get other opinions before going back and making the change.

Reply:
Agreed. Our convention is to use singular when there is a one-to-one-relation (e.g., node_at_cell) and plural when there is one-to-many (e.g., links_at_node, faces_at_cell). So in keeping with that it should be d8s_at_node.